REST/Documentation.json

```diff
@@ -10646,83 +9770,6 @@
       },
       "type": "object"
     },
-    "JobModel": {
-      "properties": {
-        "customFields": {
-          "type": "array",
-          "items": {
-            "type": "object",
-            "$ref": "#/definitions/CustomFieldModel"
-          }
-        },
-        "description": {
-          "type": "string"
-        },
-        "duration": {
-          "type": "integer"
-        },
-        "id": {
-          "type": "string"
-        },
-        "jobState": {
-          "type": "string"
-        },
-        "lat": {
-          "type": "number"
-        },
-        "lon": {
-          "type": "number"
-        },
-        "poiId": {
-          "type": "integer"
-        },
-        "routeId": {
-          "type": "string"
-        },
-        "scheduledTime": {
-          "type": "string"
-        },
-        "workerId": {
-          "type": "integer"
-        }
-      },
-      "required": [
-        "duration",
-        "lat",
-        "lon",
-        "poiId",
-        "workerId"
-      ],
-      "type": "object"
-    },
-    "RouteModel": {
-      "properties": {
-        "colour": {
-          "type": "string"
-        },
-        "customFields": {
-          "type": "array",
-          "items": {
-            "type": "object",
-            "$ref": "#/definitions/CustomFieldModel"
```

Comment:
I believe the definition for CustomFieldModel can be removed as well. Looks like this is a dispatch things only